rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -1709,9 +1710,74 @@ error_code sys_fs_fcntl(ppu_thread& ppu, u32 fd, u32 op, vm::ptr<void> _arg, u32
 		break;
 	}
 
-	case 0x00000025: // cellFsSdataOpenWithVersion
+	case 0xe0000025: // cellFsSdataOpenWithVersion
 	{
 		break;
```

Comment:
NANI

Reply:
lol

Reply:
Turns out the answer to how many Devs does it take to see a copy paste fail is 5